File: Python/palindrome.py
import re

s = "0P"

def isPalindrome (s):

    clean = re.sub(r'[^a-zA-ZáéíóúÁÉÍÓÚüÜñÑ0-9]', '', s).lower()
    flip = ""
    for x in range(len(clean) - 1, -1, -1):
        flip += clean[x]

    if flip == clean: return True
    else: return False



print( isPalindrome(s))

File: JavaScript/palindrome.js

let s = "0P"
var isPalindrome = function(s) {

    let clean = s.replace(/[^a-zA-ZáéíóúÁÉÍÓÚüÜñÑ0-9]/g, '').toLowerCase();
    let flip = ""
        for (let x = clean.length -1; x >= 0; x--) {

            flip += clean[x]
            
        }
        if ( flip == clean) return true
        else return false
    
};

console.log(isPalindrome(s))